Formula/latexml.rb

```diff
@@ -14,24 +14,277 @@ class Latexml < Formula
     sha256 "884426eb041a9fa05ba6ebc64c64f4ce76f7c10cab3c5c1b98bcce201831c9d2" => :mavericks
   end
 
+  requires_perl = []
+  requires_perl << "with-perl"
+  depends_on "perl"                             # Actually, Imagemagick already installs Perl
+  depends_on "imagemagick" => requires_perl     # Install ImageMagick with PerlMagick
```

Comment:
This should trigger an audit failure which you've managed to defeat with the variable.

Reply:
You're right, but unfortunately ImageMagick needs to be installed with its own Perl module for LaTeXML to work. I am new to Homebrew recipes; do you have any suggestion for this problem?